proto/bep/bep.proto

```diff
@@ -52,6 +52,7 @@ message Folder {
   bool ignore_delete = 5;
   bool disable_temp_indexes = 6;
   bool paused = 7;
+  bool out_of_space = 8;
```

Comment:
Instead of adding more bools, I think the `paused` field could become an enum `stopped_reason`.

```
enum StoppedReason {
  STOPPED_REASON_UNSPECIFIED = 0; // not stopped
  STOPPED_REASON_PAUSED = 1; // previous "true"
  STOPPED_REASON_OUT_OF_SPACE = 2;
}
```